utils/dwarf.c

```diff
@@ -1764,6 +1768,7 @@ static void build_dwarf_info(struct uftrace_dbg_info *dinfo, struct uftrace_symt
 	for (i = 0; i < rets->nr; i++)
 		free_filter_pattern(&ret_patt[i]);
 	free(ret_patt);
+	free(dinfo->locs);
```

Comment:
Thanks for catching these, but they are totally unrelated changes.  You should have them in a separate commit, preferably in a separate PR.

Reply:
Sure. I'll sumit a separate issue for these handler fix!